packages/orchestration/src/utils/start-helper.js

```diff
@@ -106,20 +114,38 @@ export const provideOrchestration = (
     vowTools,
   });
 
-  const facade = makeOrchestrationFacade({
-    zcf,
-    zone: zones.orchestration,
+  const makeOrchestratorKit = prepareOrchestratorKit(zones.orchestration, {
+    asyncFlowTools,
     chainHub,
-    makeLocalOrchestrationAccountKit,
+    localchain: remotePowers.localchain,
     makeRecorderKit,
-    makeCosmosOrchestrationAccount,
     makeLocalChainFacade,
     makeRemoteChainFacade,
+    storageNode: remotePowers.storageNode,
+    orchestrationService: remotePowers.orchestrationService,
+    timerService,
+    vowTools,
+    zcf,
+  });
+
+  const makeOrchestrator = pickFacet(makeOrchestratorKit, 'orchestrator');
+
+  const facade = makeOrchestrationFacade({
+    zcf,
+    zone: zones.orchestration,
+    makeRecorderKit,
+    makeOrchestrator,
     asyncFlowTools,
     vowTools,
     ...remotePowers,
   });
-  return { ...facade, chainHub, vowTools, zone: zones.contract };
+  return {
+    ...facade,
+    chainHub,
+    vowTools,
+    zoeTools,
+    zone: zones.contract,
+  };
```

Comment:
Tangential - wondering what parts of this we can make widely available in the bootstrap space. Thinking the benefits would be:
1. Smaller bundles for guest code + less code on chain
2. (marginal) less setup for devs since they can just grab a power